packages/amplify-codegen-e2e-core/src/utils/graphql-config-helper.ts

```diff
@@ -0,0 +1,51 @@
+import { AmplifyFrontendConfig, AmplifyFrontend } from './frontend-config-helper';
+
+export function constructGraphQLConfig(
```

Comment:
It looks like the field such as `apiId`, `generatedFileName` is not configurable in this function.

Reply:
Added `apiId`. `generatedFileName` depends on the frontend config being passed. It is set accordingly.